frontend/uddhar-frontend/src/coordinator/components/UserTable.jsx

```
@@ -53,19 +53,23 @@ const UserTable = () => {
+                          : user.status === "Approved"
+                          ? "bg-green-200 text-green-800"
+                          : "bg-red-200 text-red-800"
+                      }`}
                     >
                       {user.status}
                     </p>
                   </td>
                   <td>
-                    <button className="border-2 rounded-[5px] bg-green-200 p-1 cursor-pointer text-green-500 mr-1">
-                      ✔️
+                    <button className="border-2 rounded-[5px] bg-green-400 px-2 py-1 cursor-pointer text-black mr-1">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)